runtime/pyproject.toml

```diff
@@ -2,10 +2,11 @@
 requires = [
     "setuptools>=42",
     "wheel",
-    "cmake",
+    "cmake==3.25.0",
+    "nanobind==1.4.0",
```

Comment:
could this be `>=1.4.0` just like we had for pybind ? Mostly to avoid minor number variations across all OSes

Reply:
nanobind is a pure Python package so should never have availability issues. However, making it like what we do for pybind is fine with me and I've switched it.